packages/fxa-content-server/app/styles/_modules.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../node_modules/jquery-modal/jquery.modal';
+@import '~jquery-modal/jquery.modal';
```

Comment:
`~` is a special sass-loader flag to let it know to resolve the import externally